pkg/graveler/graveler.go

```diff
@@ -912,6 +912,11 @@ type CommittedManager interface {
 
 	// GetRangeIDByKey returns the RangeID that contains the given key.
 	GetRangeIDByKey(ctx context.Context, ns StorageNamespace, id MetaRangeID, key Key) (RangeID, error)
+
+	// Import applies changes from 'source' to 'destination' and returns the ID of the new metarange.
+	// This is similar to the Merge operation, but it will override existing prefixes in the destination
+	// if they also exist in the source.
+	Import(ctx context.Context, ns StorageNamespace, destination, source MetaRangeID) (MetaRangeID, error)
```

Comment:
Not going to work since you need the context of the request. If the source contains just `foo/bar.txt`, would you replace the prefix `` (empty prefix), the prefix `foo/`, the path `foo/bar.txt` or maybe the prefix `f`??

Reply:
As Itai stated, you'll need to pass a list of the source prefixes here

Reply:
Also, please note the differentiation between source types object and prefix